tool/lib/utils.dart

```diff
@@ -122,3 +124,37 @@ Future<void> runAll(
     );
   }
 }
+
+String pathFromRepoRoot(String pathFromRoot) {
+  return path.join(DevToolsRepo.getInstance()!.repoPath, pathFromRoot);
+}
+
+extension DevtoolsProcess on Process {
```

Comment:
Instead of creating another way to run a process, can we just use the `runProcess` helper from above?

Reply:
oh wow. I totally didn't see that helper. I tried searching `Process.run` to see if we already had one. Thanks for pointing this out!

Reply:
Using the ProcessManager automatically streams the process output to stdout and stderr, which is nice